pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -265,6 +267,8 @@ const (
 	NodeGroupTypeManaged NodeGroupType = "managed"
 	// NodeGroupTypeUnmanaged defines an unmanaged nodegroup
 	NodeGroupTypeUnmanaged NodeGroupType = "unmanaged"
+	// NodeGroupTypeUnowned defines an unowned nodegroup
+	NodeGroupTypeUnowned NodeGroupType = "unowned"
```

Comment:
this is always also managed, right?

Reply:
yes @michaelbeaumont . I was tempted to call it `NodeGroupTypeManagedUnowned` but it felt like too much 😂

Reply:
Maybe use the comment to elaborate?

Reply:
i was just thinking it might be unclear (at first glance) to users if they see `Unowned` whether it's managed or unmanaged.